tests/eval_files/noqa_no_autofix.py

```diff
@@ -9,22 +8,13 @@
 async def foo() -> Any: ...
 
 
-async def foo_no_noqa_102():
-    try:
-        pass
-    finally:
-        await foo()  # ASYNC102: 8, Statement("try/finally", lineno-3)
+async def foo_no_noqa_109(timeout):  # ASYNC109: 26, "trio"
+    ...
 
 
-async def foo_noqa_102():
-    try:
-        pass
-    finally:
-        await foo()  # noqa: ASYNC102
+async def foo_noqa_102(timeout):  # noqa: ASYNC109, "trio"
```

Comment:
```suggestion
async def foo_noqa_102(timeout):  # noqa: ASYNC109
```

noqa is not a magic marker for the test infra, it's handled by visitors, which don't care about anything other than the code.